statsmodels/stats/proportion.py

```diff
@@ -123,6 +123,183 @@ def func(qi):
         raise NotImplementedError('method "%s" is not available' % method)
     return ci_low, ci_upp
 
+def multinomial_proportions_confint(counts, alpha=0.05, method='goodman'):
+    '''Confidence intervals for multinomial proportions.
+
+    Parameters
+    ----------
+    counts : array_like of int
+        Number of observations in each category.
+    alpha : float in (0, 1), optional
+        Significance level, defaults to 0.05.
+    method : {'goodman', 'sison_glaz'}, optional
+        Method to use to compute the confidence intervals; available methods
+        are:
+
+         - `goodman`: based on a chi-squared approximation, valid if all
+           values in `counts` are greater or equal to 5 [2]_
+         - `sison_glaz`: less conservative than `goodman`, but only valid if
+           `counts` has 7 or more categories (``len(counts) >= 7``) [3]_
+
+    Returns
+    -------
+    array of [float, float] arrays
+        Lower and upper confidence levels for each category, such that overall
+        coverage is (approximately) `1-alpha`.
+
+    Raises
+    ------
+    ValueError
+        If `alpha` is not in `(0, 1)` (bounds excluded), or if the values in
+        `counts` are not all positive or null.
+    NotImplementedError
+        If `method` is not kown.
+    Exception
+        When ``method == 'sison_glaz'``, if for some reason `c` cannot be
+        computed; this signals a bug and should be reported.
+
+    Notes
+    -----
+    The `goodman` method [2]_ is based on approximating a statistic based on
+    the multinomial as a chi-squared random variable. The usual recommendation
+    is that this is valid if all the values in `counts` are greater than or
+    equal to 5. There is no condition on the number of categories for this
+    method.
+
+    The `sison_glaz` method [3]_ is based on a normal approximation that
```

Comment:
Talking about what allows them to go from eq. 6 to eq. 7 (page 2 of the Sison & Glaz paper). It's a "Law of large numbers" approximation ("... [the two distributions] converge to the same multivariate normal distribution ..."), so not sure how to call it. Could be "a distribution approximation that converges ...".